### ISSUE TITLE
Hazard examples repeat info in a summary

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#748]
A lot of the hazard definitions appear to be violating the principle of not repeating machine-readable info in the summary. It's correct when there's a hazard to state what it is, but the unknown and non-hazards terms repeat the example explanatory text both in a comment and in the summary. For example:

```
This value is declared in the package.opf metadata after an author or publisher has audited the content (including all videos and animations) and confirmed that no flashing hazards are present.

...

<!-- This publication has been verified to be free of flashing hazards. -->
<meta property="schema:accessibilityHazard">noFlashingHazard</meta>

<meta property="schema:accessibilitySummary">This publication is free of known flashing hazards.</meta>
```

I'd suggest keeping only the explanatory text and accessibilityHazard declarations and dropping the comments and summaries.
